### PR TITLE
Rewrite gRPC service invocation example

### PR DESCRIPTION
**Pull request by MregXN**
Please reference the issue: #682

### CODE REVIEW

File: examples/src/main/java/io/dapr/examples/invoke/grpc/README.md [hunk 1]
```diff
@@ -95,55 +93,59 @@ The `app-id` argument is used to identify this service in Dapr's runtime. The `a
...
-  - '== APP == Message sent: Message #1'
+  - '== APP == INFO: Will try to greet World ...'
+  - '== APP == INFO: Greeting: Hello World'
 background: true
 sleep: 10
 -->
 
 ```bash
-dapr run --app-id invokegrpc -- java -jar target/dapr-java-sdk-examples-exec.jar io.dapr.examples.invoke.grpc.HelloWorldClient
+dapr run --app-id invokegrpc --app-protocol grpc --dapr-grpc-port 50051 -- java -jar target/dapr-java-sdk-examples-exec.jar io.dapr.examples.invoke.grpc.HelloWorldClient
```
Comment (artursouza) [inline examples/src/main/java/io/dapr/examples/invoke/grpc/README.md:148]: No need to set the port, CLI picks one automatically.

File: examples/src/main/java/io/dapr/examples/invoke/grpc/README.md [hunk 2]
```diff
@@ -95,55 +93,59 @@ The `app-id` argument is used to identify this service in Dapr's runtime. The `a
...
-
-        Thread.sleep(1000);
-
-        // This is an example, so for simplicity we are just exiting here.  
-        // Normally a dapr app would be a web service and not exit main.
-        System.out.println("Done");
+
+    String user = "World";
+    // Access a service running on the local machine on port 50051
+    String target = "localhost:50051";
```
Comment (artursouza) [inline examples/src/main/java/io/dapr/examples/invoke/grpc/README.md:105]: Consume from DAPR_GRPC_PORT env that is injected automatically and fallback to the default port.

**Review by artursouza — CHANGES_REQUESTED** [review]

**Comment by artursouza** [comment]
Thanks. Just some minor changes.

File: examples/src/main/java/io/dapr/examples/invoke/grpc/README.md (hunk 2 above)
Comment (MregXN) [reply examples/src/main/java/io/dapr/examples/invoke/grpc/README.md:105]: Modified, Thanks.

File: examples/src/main/java/io/dapr/examples/invoke/grpc/README.md (hunk 1 above)
Comment (MregXN) [reply examples/src/main/java/io/dapr/examples/invoke/grpc/README.md]: Modified, Thanks.

**Comment by cicoyle** [comment]
@MregXN - mind merging in master? I'm unable to, otherwise I would. Then this looks like it can be merged

**Comment by MregXN** [comment]
> @MregXN - mind merging in master? I'm unable to, otherwise I would. Then this looks like it can be merged

yes it is ready to mrege.  @artursouza  @mukundansundar

**Review by cicoyle — APPROVED** [review]
LGTM. Thanks @MregXN

**Review by artursouza — APPROVED** [review]

**Pull request merged**